tikv-configuration-file.md

```diff
@@ -1321,6 +1321,25 @@ Configuration items related to TiCDC.
 + Default value: `6`, which means 6 tasks can be concurrent executed at most.
 + Note: The value of `incremental-scan-concurrency` must be greater than or equal to that of `incremental-scan-threads`; otherwise, TiKV will report an error at startup.
 
+## resolved-ts
+
+Configuration items related to maintaining the Resolved TS to serve Stale Read requests.
+
+### `enable`
+
++ Enables whether to maintain the Resolved TS for all Regions.
++ Default value: `false`
+
+### `advanced-ts-interval`
+
++ The time interval for regularly advancing the Resolved TS.
```

Comment:
```suggestion
+ The time interval at which the Resolved TS is processed.
```

Reply:
Perhaps we can reuse the description of `cdc.min-ts-interval`, they are used for the same goal.